compiler/toEcmascript.test.js

```diff
@@ -192,8 +192,8 @@ popSourceLocation({
 return {};
 
 } catch (error) { throw error; }
-`
-  );
+`,
+  ]);
 });
 
 test('Bind await to calls properly.', async (t) => {
```

Comment:
What's going on here with the aaaaa? 😛

Reply:
There was some weird issue about excessively long lines, I think.